proposals/incubation/cert-manager.md

```diff
@@ -0,0 +1,138 @@
+# cert-manager Incubation Proposal
+
+cert-manager joined the CNCF as a sandbox project just over a year ago, and we firmly believe that the project is ready to take the next step and graduate to the Incubation stage.
+
+cert-manager provides reliable, automated Kubernetes-native management of X.509 certificates. It reduces the risk of certificate-related outages and enables PKI at scale, and is treated by many teams as de-facto part of the core components of a Kubernetes or OpenShift deployment.
+
+The project comprises cert-manager itself - which adds Kubernetes Custom Resource Definitions (CRDs) for certificate-related concepts, and provides controllers to manage them - along with a selection of subprojects which focus on solving other issues relating to certificate management.
+
+The main cert-manager repo has 8.4k stars on GitHub and hundreds of projects downstream which depend on cert-manager code in some way. Thousands of people have contributed to the project in some way and cert-manager's use and prominence remains strong.
+
+## Incubation Criteria Checklist
+
+Based on the [graduation criteria][GraduationCriteria], cert-manager seems to clearly tick the boxes required for incubation:
+
+### Production Use
+
+> Document that it is being used successfully in production by at least three independent end users
+
+cert-manager is used in production by _many_ more than three independent end users. Several major cloud native projects including [Istio][IstioCertManager], the [Kubernetes Cluster API][ClusterAPICertManager] and [Knative][KnativeCertManager] either suggest or require that cert-manager be installed.
```

Comment:
fwiw, ArgoCD uses cert-manager for its own demo instance: https://cd.apps.argoproj.io/applications/cert-manager

(We certainly use cert-manager for our k8s clusters, we've occasionally made PRs to list ourselves as a consumer of projects, but haven't generally. I'm not personally sure how I feel about such links...)

Reply:
Thanks for the heads up! I'll add the argo link, great to have another CNCF example.

If you end up wanting to raise a PR for users.md please do feel free :grin: